### ISSUE TITLE
Escape Key and Command Line broken when editing SQL Server Unit Tests in VS 2015

### ISSUE THREAD

**Issue by freakingawesome** [jaredpar/VsVim#1689]
Create a new SQL Server Unit Test, then edit the SQL text. Lots of VsVim functionality works fine, but two major things are broken:

1. The Escape key won't work. It makes no impact on state from any mode. `Ctrl-[` still works.
2. You can get to the command line by typing `:`, but pressing Return or Escape does nothing.

It looks like the default editor for SQL Server Unit Tests is "CSharp Form Editor".

**Comment by jaredpar** [comment]
Thanks for reporting.

Unfortunately this is a known issue.  The SQL Server Unit Test editor hijacks the keyboard in a very non-standard (and very unsupported) manner.  Any key that it wants to have an action for, like <kbd>Esc</kbd>, never makes it to extensions.  

I've spent several hours looking at this editor in the past and unfortunately there is no way I can discern to get VsVim back into the keyboard loop here.  Additionally there is nothing about the editor that stands out so that I could just disable VsVim entirely here either.  It's a very frustrating issue.  

Closing as a dupe of #808.
